### DISCUSSION TITLE
vMix and TallyTec.

### DISCUSSION THREAD

**Discussion by Petrovski1504** [josephdadams/TallyArbiter#709] [Q&A]
Hi!

I have a PC with vMix and another PC that I use to access the vMix web interface for graphics control and text input for graphics. Both PCs are located in the studio's datacenter, and the equipment is connected on-site using two KVMs from BlackBox. The KVM of the PC with vMix has two Stream Decks connected to it as control panels, and I have thus reached the maximum connection limit for this KVM.

My question is whether I can take the Tally signal from the second PC (the one I use to access the vMix browser interface) and forward it, and if so, how can I do this via TallyArbiter?

In this studio, I use TallyTec equipment for Tally signaling so I think I have to buy the Convertor Box to connect it between the second PC and TallyTec.

Thanks!

**Comment by josephdadams** [comment]
I think you could put TA directly on the vmix computer l and use the vmix tcp api that TA supports for tally. Once it is coming into TA, you can do whatever you need to with it there. I don’t know anything about tallytec systems.
